src/ai.ts

```diff
@@ -1,6 +1,7 @@
 import { generateObject } from "ai";
 import { createAnthropic } from "@ai-sdk/anthropic";
 import { createOpenAI } from "@ai-sdk/openai";
+import { createGoogleGenerativeAI } from "@ai-sdk/google";
```

Comment:
Consider adding error handling for the `createGoogleGenerativeAI` function to ensure that any issues during the instantiation of the model are properly managed.